generated/nidmm/session.py

```diff
@@ -986,7 +986,7 @@ def configure_frequency_voltage_range(self, voltage_range):
         measurements.
 
         Args:
-            voltage_range (ViReal64):
+            voltage_range (float):
```

Comment:
I like this better. The Python developer shouldn't need to know or care about ViReal64.